content/10-managing-databases/07-introduction-database-caching.mdx

```diff
@@ -0,0 +1,112 @@
+
+### Performance
+As touched on briefly, database caching improves the performance of a database by making data more easily accessed. The cache acts has a sort of “keyboard short-cut” or “hot-key” for the application to reference data that it frequently is calling upon. 
+
+This speedier request can minimize the workload of the database, keeping it from spending inefficient amounts of time doing repetitive tasks. Instead making these tasks more efficient and simplifying data access. 
+
+### Availability
+While not a 100% failover strategy, caching also provides benefits to overall database [availability](/intro/database-glossary#availability). Depending on where the cache is stored, the cache can still provide a place for the application to call upon for data in the case of the primary database server becoming unavailable for any reason.
+
+While database performance is generally the primary reason for adopting a caching strategy, you also have the added benefit of some additional resiliency in the case of any backend failures. 
+
+### Scalability
+Similarly to added high availability, database caching has a positive effect on [scalability](/intro/database-glossary#scaling). While it shouldn’t be your main consideration for a database scaling strategy, implementing caching to improve database performance reduces your database workload, therefore distributing backend queries across entities. 
+
+This distribution lightens the load on a primary database and can reduce costs and provide more flexibility in the processing of your data. This result alleviates the need to scale and does more with the resources you already have on hand, potentially pushing the need to scale into the future.
+
+## What are the different database caching strategies?
+
+Before adopting database caching into your data access flow, it is important to consider which caching strategy is best suited for the job. For any scenario, the relationship between the database and the cache can have a different impact on performance and system structure. Planning ahead and considering all options will lead to fewer headaches down the road. 
+
+The five most popular strategies to consider are cache-aside, read-through, write-through, write-back, and write-around. We’ll cover the data source to cache relationship and process of each strategy.
+
+### Cache-aside
+
+In a cache-aside arrangement, the database cache sits next to the database. When the application requests data, it will check the cache first. If the cache has the data (a cache hit), then it will return it. If the cache does not have the data (a cache miss), then the application will query the database. The application then stores that data in the cache for any subsequent queries.
+
+![Cache-Aside](/content/database-caching/cache-aside.png)
```

Comment:
Uh this one is confusing. 1 I would have expected a request from the app to the cache, 2 then a response that nothing found back, then 3 a query to the database, 4 a response back, then 5 a write to the cache as last step.